src/fetch/payouts.py

```diff
@@ -321,17 +344,18 @@ def construct_payout_dataframe(
     # TODO - After CIP-20 phased in, adapt query to return `solver` like all the others
     slippage_df = slippage_df.rename(columns={"solver_address": "solver"})
     # 1. Assert existence of required columns.
-    validate_df_columns(payment_df, slippage_df, reward_target_df)
+    validate_df_columns(payment_df, slippage_df, reward_target_df, mev_kickback_df)
 
     # 2. Normalize Join Column (and Ethereum Address Field)
     join_column = "solver"
-    normalize_address_field(payment_df, join_column)
-    normalize_address_field(slippage_df, join_column)
-    normalize_address_field(reward_target_df, join_column)
-
-    # 3. Merge the three dataframes (joining on solver)
-    merged_df = payment_df.merge(slippage_df, on=join_column, how="left").merge(
-        reward_target_df, on=join_column, how="left"
+    for data_frame in [payment_df, slippage_df, reward_target_df, mev_kickback_df]:
+        normalize_address_field(data_frame, join_column)
+
+    # 3. Merge the four dataframes (joining on solver)
+    merged_df = (
+        payment_df.merge(slippage_df, on=join_column, how="left")
+        .merge(reward_target_df, on=join_column, how="left")
+        .merge(mev_kickback_df, on=join_column, how="left")
```

Comment:
Nit: If we're okay with importing `from functools import reduce` then we can tidy this up even further:

```suggestion
    data_frames = [payment_df, slippage_df, reward_target_df, mev_kickback_df]
    for data_frame in data_frames:
        normalize_address_field(data_frame, join_column)

    # 3. Merge the four dataframes (joining on solver)
    merged_df = reduce(lambda left, right: pd.merge(left, right, on=join_column, how='left'), data_frames)
```

Reply:
REALLY! Who knew? -- you knew!